package.json

```
@@ -14,6 +14,7 @@
     "storybook:clean": "start-storybook -p 6006 --no-manager-cache",
     "build": "npx lerna run build",
     "build-react": "cd packages/react && yarn build",
+    "pack-react": "cd packages/react && yarn pack",
```

We should use `yarn workspace` for this. Also suggested name, to conform to other scripts.
```suggestion
    "pack:react": "yarn workspace @digdir/design-system-react pack",
```

---

alright, i tried to conform to the one directly above 😅